pkg/bindings/nats/errors.go

```diff
@@ -0,0 +1,5 @@
+package nats
+
+import "fmt"
+
+var BinaryEncodingNotSupported = fmt.Errorf("binary encoding for NATS protocol is not supported")
```

Comment:
You don't need it, if you need to signal that you don't support the binary encoding, you must use `binding.ErrNotBinary`